data/json/items/comestibles/med.json

```diff
@@ -1313,6 +1318,7 @@
     "fun": 25,
     "addiction_potential": 25,
     "addiction_type": "opiate",
+    "vitamins": [ [ "vit_mme", 3000 ] ],
```

Comment:
if it's 10mg syrette as the comment implies, then it should be 10 MME which is 1000 vit_mme?

Reply:
Per Venera injectable Opiates are triple the MME of oral equivalents.